flow/project.py

```diff
@@ -2630,6 +2630,7 @@ class MyProject(FlowProject):
             '{} options'.format(self._environment.__name__))
         self._environment.add_args(env_group)
         parser_submit.set_defaults(func=self._main_submit)
+        print('Using environment configuration:',self._environment.__name__)
```

Comment:
*(You will need to add a ' ' (space) behind the comma to avoid a style violation.)*

Reply:
Fixed in commit 0218228, thanks!